context/AuthContext.tsx

```diff
@@ -0,0 +1,111 @@
+import {createContext, useContext, useEffect, useState} from "react"
```

Comment:
Change this to `{createContext, useContext, useEffect, useState}, React from "react"` or `{createContext, useContext, useEffect, useState, ReactNode} from "react"`; FYI, you are attempting to use `React.ReactNode` below, which requires this import

Reply:
resolved in additional commit FYI